app/shared_features/set_card/analysis/SetAnalysis.js

```diff
@@ -23,14 +23,26 @@ class SetAnalysis extends Component {
             } else {
                 var text = description;
             }
-            return (
-                <View style={{flex: 1, marginTop: -2, justifyContent: 'center'}}>
-                    <Text style={styles.text}>
-                        {value}
-                    </Text>
-                    <Text style={styles.metric}>{text}</Text>
-                </View>  
-            );
+
+            if (!this.props.rpe && (text === 'RPE' || text === 'RPE1RM')) {
+                return (
+                    <View style={{flex: 1, marginTop: -2, justifyContent: 'center'}}>
+                        <Text style={styles.redText}>
+                            {value}
+                        </Text>
+                        <Text style={styles.redMetric}>{text}</Text>
+                    </View>  
+                );
+            } else {
+                return (
+                    <View style={{flex: 1, marginTop: -2, justifyContent: 'center'}}>
+                        <Text style={styles.text}>
+                            {value}
+                        </Text>
+                        <Text style={styles.metric}>{text}</Text>
+                    </View>  
+                );
+            }
```

Comment:
- [ ] quick refactor, make the text style a variable and then use it in the View instead of a giant if statement with 2 views